### DISCUSSION TITLE
Now CLI deployment auto-cancels -- why?

### DISCUSSION THREAD

**Discussion by morgoe** [vercel/vercel#4220] [General]
I'm having a ton of issues with deployment.

First the deployments were all queued in a massive backlog and stayed that way for over an hour.

Second, the deployment went ahead, but the GitHub integration somehow used the wrong environment variables, so currently my live production site is using staging data. I'm **urgently** trying to fix this.

Third, I'm now trying to deploy via the command line to fix this, and the deployment auto cancels. How do I resolve this?

<img width="469" alt="Screen Shot 2020-05-01 at 6 55 50 pm" src="https://user-images.githubusercontent.com/3588946/80794496-65ca2000-8bdd-11ea-990e-c13a7f36c299.png">

**Comment by coetry** [comment]
Hi @morgoe I'm really sorry that you're running into this and we're more than happy to take a closer look. Can you email us at support@vercel.com so we can look into the specifics of your case?
